pkgs/dividat-driver/default.nix

```diff
@@ -1,18 +1,35 @@
-{stdenv, fetchurl}:
-stdenv.mkDerivation rec {
-  name = "dividat-driver-${version}";
-  version = "2.1.0";
-  channel = "master";
-
-  src = fetchurl {
-    url = "https://dist.dividat.com/releases/driver2/${channel}/${version}/dividat-driver-linux-amd64-${version}";
-    sha256 = "0f5hd7mxrhsaxmwdyys1vk9z5rxm57y3w7qjlqj7l5v2v41g0vrm";
+{ stdenv, fetchFromGitHub, pkgs, buildGoModule }:
+
+let
+
+  channel = "develop";
+
+  version = "unstable";
```

Comment:
Should we create a new version on the driver ?

Reply:
Maybe it's time for [`2.2.0-rc2`](https://github.com/dividat/driver/releases/tag/2.2.0-rc1). We have yet to settle on a future release model for the driver, now that its typical delivery is baked into PlayOS.

Reply:
I tagged the last commit of `develop` with [2.2.0-rc2](https://github.com/dividat/driver/releases/tag/2.2.0-rc2).

I am wondering if, in the future, the driver is only shipped with PlayOS, that it should be part of the PlayOS repository.

Reply:
> I am wondering if, in the future, the driver is only shipped with PlayOS, that it should be part of the PlayOS repository.

I think it's still useful to have a separate project repo. We use it independently for development, and outside collaborators will continue to use it independently, too.